x-pack/plugins/cases/server/routes/api/cases/sub_case/get_sub_case.ts

```diff
@@ -32,44 +30,15 @@ export function initGetSubCaseApi({ caseService, router, logger }: RouteDeps) {
         const soClient = context.core.savedObjects.getClient({
           includedHiddenTypes: SAVED_OBJECT_TYPES,
         });
-        const includeComments = request.query.includeComments;
 
-        const subCase = await caseService.getSubCase({
-          soClient,
-          id: request.params.sub_case_id,
-        });
-
-        if (!includeComments) {
-          return response.ok({
-            body: SubCaseResponseRt.encode(
-              flattenSubCaseSavedObject({
-                savedObject: subCase,
-              })
-            ),
-          });
-        }
-
-        const theComments = await caseService.getAllSubCaseComments({
-          soClient,
-          id: request.params.sub_case_id,
-          options: {
-            sortField: 'created_at',
-            sortOrder: 'asc',
-          },
-        });
+        const client = await context.cases.getCasesClient();
```

Comment:
Same as before `soClient` shouldn't be part of the arguments. The `CasesClient` provide it to all sub clients.